Detectors/MUON/MCH/Base/include/MCHBase/Digit.h

```diff
@@ -47,13 +47,16 @@ class Digit
 
   Digit() = default;
 
-  Digit(int detid, int pad, unsigned long adc, Time time);
+  Digit(int detid, int pad, unsigned long adc, Time time, uint16_t nSamples);
```

Comment:
I would give this additional parameter a default of one for the moment, so we don't have to touch all the simulation parts as of now. What do you think ?

Reply:
@aphecetche Ok, I force-pushed a new version that leaves the simulation code untouched.